.packit.yaml

```diff
@@ -0,0 +1,45 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-smart_proxy_pulp.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-smart_proxy_pulp.spec
```

Comment:
Mimd fixing the indenting here?

Reply:
If you update the template ;)

Reply:
https://github.com/theforeman/smart_proxy_plugin_template/pull/2